src/main/index.ts

```
@@ -187,3 +200,28 @@ ipcMain.handle('get-series-charCount-list', (_, date: { year: number; month: num
 
   return charCounts[year][month]
 })
+
+function readDirectoryRecursive(dirPath: string): any[] {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)